### DISCUSSION TITLE
How to fix Join not a function issue

### DISCUSSION THREAD

**Discussion by virginus01** [vercel/next.js#58029] [Help]
### Summary

How do I get rid of this:

`TypeError: path__WEBPACK_IMPORTED_MODULE_1___default(...).join is not a function`

**Comment by sefin23** [comment]
Hi,

This happens when you import path the wrong way in a Next.js (webpack) environment. The default export doesn't work — you need to destructure it.

Change this:

js
```javascript
import path from 'path'
path.join(...)

``` 

To this:

js
```javascript
const path = require('path')
path.join(...)
``` 


Or if you're using ESM:

js
```javascript
import { join } from 'path'
join(...)
``` 

That should fix it!
